src/data.c

```diff
@@ -3788,6 +3788,8 @@ syms_of_data (void)
 {
   Lisp_Object error_tail, arith_tail;
   rust_init_syms();
+  DEFSYM (QCnull, ":null");
+  DEFSYM (QCfalse, ":false");
```

Comment:
I can remove this code and add calls to `def_lisp_sym!` once I bring that over from the remacs codebase.